### PR TITLE
Replace vbNewLine with Environment.NewLine

### PR DESCRIPTION
**Pull request by Youssef1313**
vbNewLine is now obsolete.

@gewarren FYI

### CODE REVIEW

File: docs/visual-basic/language-reference/constants-and-enumerations.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Visual Basic supplies a number of predefined constants and enumerations for deve
 |`vbCrLf`|Carriage return/linefeed character combination.|  
 |`vbCr`|Carriage return character.|  
 |`vbLf`|Linefeed character.|  
-|`vbNewLine`|Newline character.|  
+|`vbNewLine`|Newline character. This is obsolete starting with .NET 5.0|
```
Comment (Youssef1313) [inline docs/visual-basic/language-reference/constants-and-enumerations.md:37]:
```suggestion
|`vbNewLine`|Newline character. This is obsolete starting with .NET Core 3.0|
```

File: docs/visual-basic/language-reference/runtime-library-members.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ The `Microsoft.VisualBasic` namespace contains the classes, modules, constants,
 |<xref:Microsoft.VisualBasic.Constants.vbLinguisticCasing>|<xref:Microsoft.VisualBasic.Constants.vbLong>|<xref:Microsoft.VisualBasic.Constants.vbLongDate>|<xref:Microsoft.VisualBasic.Constants.vbLongTime>|  
 |<xref:Microsoft.VisualBasic.Constants.vbLowerCase>|<xref:Microsoft.VisualBasic.Constants.vbMaximizedFocus>|<xref:Microsoft.VisualBasic.Constants.vbMethod>|<xref:Microsoft.VisualBasic.Constants.vbMinimizedFocus>|  
 |<xref:Microsoft.VisualBasic.Constants.vbMinimizedNoFocus>|<xref:Microsoft.VisualBasic.Constants.vbMonday>|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxHelp>|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxRight>|  
-|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxRtlReading>|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxSetForeground>|<xref:Microsoft.VisualBasic.Constants.vbNarrow>|<xref:Microsoft.VisualBasic.Constants.vbNewLine>|  
+|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxRtlReading>|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxSetForeground>|<xref:Microsoft.VisualBasic.Constants.vbNarrow>|<xref:Microsoft.VisualBasic.Constants.vbNewLine> (This is obsolete starting with .NET 5.0)|
```
Comment (Youssef1313) [inline docs/visual-basic/language-reference/runtime-library-members.md:55]:
```suggestion
|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxRtlReading>|<xref:Microsoft.VisualBasic.Constants.vbMsgBoxSetForeground>|<xref:Microsoft.VisualBasic.Constants.vbNarrow>|<xref:Microsoft.VisualBasic.Constants.vbNewLine> (This is obsolete starting with .NET Core 3.0)|
```

File: docs/visual-basic/reference/command-line-compiler/vbruntime.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ Specifies that the compiler should compile without a reference to the Visual Bas
   
 - <xref:Microsoft.VisualBasic.Constants.vbLf> constant  
   
-- <xref:Microsoft.VisualBasic.Constants.vbNewLine> constant  
+- <xref:Microsoft.VisualBasic.Constants.vbNewLine> constant (This is obsolete starting with .NET 5.0)
```
Comment (Youssef1313) [inline docs/visual-basic/reference/command-line-compiler/vbruntime.md:65]:
```suggestion
- <xref:Microsoft.VisualBasic.Constants.vbNewLine> constant (This is obsolete starting with .NET Core 3.0)
```

**Comment by BillWagner** [comment]
Thanks @Youssef1313 

Our CI build is detecting that some of the VB code isn't in a project, so isn't verified. Can you add project files in those folders so they get validated? Thank you.

**Comment by BillWagner** [comment]
@Youssef1313 Are you going to come back to this PR and add a project for the CI builds? 

If not, we'll close this PR.

**Comment by BillWagner** [comment]
closing as stale.

**Pull request closed without merging**